packages/components/src/Table/Row/TableRow.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import TableCell from '../Cell/TableCell';
+import theme from './TableRow.scss';
+
+export function getRowId(rowDataGetter, element, index) {
+	if (rowDataGetter && rowDataGetter.getElementId) {
+		return rowDataGetter.getElementId(element);
+	} else if (element.id && typeof element.id === 'string') {
+		return element.id;
+	}
+	return index.toString();
+}
+
+function getRowClassName(classnames, index) {
```

Comment:
This can be easier to write with lodash/get
```
import get from 'lodash/get';

function getRowClassName(classnames, index) {
	return get(classnames, `rows[${index}]`);
}

```